src/api/css.ts

```
@@ -218,3 +218,125 @@ function parse(styles: string): Record<string, string> {
+    }
+    if (
+      urlDepth === 0 &&
+      (ch === 34 /* Double quote */ || ch === 39) /* Single quote */
+    ) {
+      quote = styles[i];
+      continue;
+    }
+    if (ch === 40 /* Opening parenthesis */) {
+      depth++;
+      // `url(` starts a URL token unless the value is a quoted string,
+      // which the quote branch above handles as an ordinary string.
```

ESLint

[error] Expected a block comment instead of consecutive line comments.

(multiline-comment-style)